src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -866,6 +881,9 @@ def call(
                 - 0 for tokens that are **masked**.
 
                 [What are attention masks?](../glossary#attention-mask)
+            position_ids (`tf.Tensor` of shape `(batch_size, sequence_length)`, *optional*):
+                Indices of positions of each decoder input sequence tokens in the position embeddings. Selected in the
+                range `[0, config.max_position_embeddings - 1]`.
```

Comment:
Could we make the docstring more clear to mention something like:

if `past_key_values` is passed, we also require `position_ids` to be passed.

Reply:
(same for `BART_INPUTS_DOCSTRING`)